packages/shared/routes/setup/protect/Protect.svelte

```diff
@@ -64,7 +63,19 @@
                         throw new Error('Invalid pin code!')
                     }
 
-                    await Electron.PincodeManager.set(get(activeProfile)?.id, pin)
+                    if (mobile) {
+                        const { PincodeManager } = await import('Mobile/lib/pincodeManager')
```

Comment:
I am a bit skeptical about this approach. This would create a lot of extra code in many other places. For now it is fine but the ideal way (imo) is to expose a common interface for stuff like this for mobile and desktop.